app/src/main/res/layout/player_controls.xml

```diff
@@ -128,11 +128,25 @@
                 android:contentDescription="Settings"
                 android:onClick="openTracksSheet"
                 android:src="@drawable/ic_video_settings_20dp"
-                app:layout_constraintLeft_toRightOf="@id/cycleDecoderBtn"
+                app:layout_constraintLeft_toRightOf="@id/chaptersBtn"
                 app:layout_constraintRight_toLeftOf="@id/player_overflow"
                 app:layout_constraintTop_toTopOf="@id/player_overflow"
                 app:tint="?attr/colorOnPrimarySurface" />
 
+            <ImageButton
+                android:id="@+id/chaptersBtn"
+                android:layout_width="50dp"
+                android:layout_height="50dp"
+                android:background="?attr/selectableItemBackground"
+                android:contentDescription="Settings"
+                android:onClick="pickChapter"
+                android:src="@drawable/ic_bookmark_24dp"
```

Comment:
should be some other icon to avoid confusion with bookmarking, idk what, maybe look at https://fonts.google.com/icons

Reply:
I thought about using [VLC's icon](https://github.com/videolan/vlc-android/blob/3.5.x/application/resources/src/main/res/drawable/ic_chapter.xml) for the button and I need your opinion for that

Reply:
though i like [this one](https://fonts.google.com/icons?selected=Material%20Icons%3Aplay_lesson%3A) more tbh

Reply:
Do you have a screenshot? I'm not fluent in svg paths :sweat_smile:

Reply:
![image](https://github.com/aniyomiorg/aniyomi/assets/54363735/f1eb14ab-63ab-4bfe-859a-097de2ae452f)